apis/Google.Cloud.Firestore/Google.Cloud.Firestore.Tests/Proto/test.proto

```diff
@@ -5,10 +5,16 @@ syntax = "proto3";
 package tests;
 
 option csharp_namespace = "Google.Cloud.Firestore.Tests.Proto";
+option java_package = "com.google.cloud.firestore.conformance";
```

Comment:
This is slightly unexpected in a C# repo. Is it correct? Is this also going to be used for Java tests?

Reply:
The proto is copied (by script) from google-cloud-common